fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PushDownPredicateScanRule.java

```diff
@@ -63,6 +63,8 @@ public class PushDownPredicateScanRule extends TransformationRule {
             new PushDownPredicateScanRule(OperatorType.LOGICAL_JDBC_SCAN);
     public static final PushDownPredicateScanRule BINLOG_SCAN =
             new PushDownPredicateScanRule(OperatorType.LOGICAL_BINLOG_SCAN);
+    public static final PushDownPredicateScanRule VIEW_SCAN =
+            new PushDownPredicateScanRule(OperatorType.LOGICAL_VIEW_SCAN);
```

Comment:
Why pushdown predicate into the `ViewScan` ? As my understanding if predicate got pushdown down and change plan, the MV could not be correctly rewritten ?

Reply:
it has to call PushDownPredicateScanRule rule for view scan, or it will be a pattern Filter - Scan, which is not supported.
 PushDownPredicateScanRule has been applied to table scans of all types.